test/support/assets/migrations/20111117063700-createPerson.es6.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+export default {
+
```

Comment:
Is this line break personal taste or a common thing in ES6 land? I'd prefer the removal of the padding lines in a block.